core/trino-spi/pom.xml

```diff
@@ -40,6 +40,12 @@
             <artifactId>jol-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>units</artifactId>
+            <scope>provided</scope>
```

Comment:
@pr-reviewers : Is it okay to have scope provided?
If not added then the following error happens.
<img width="1351" alt="Screen Shot 2022-09-14 at 3 44 35 PM" src="https://user-images.githubusercontent.com/6665626/190585529-284ce7a4-5ddd-442e-9831-68f4a4b7705f.png">